.github/workflows/run_secret_scan.yaml

```diff
@@ -0,0 +1,26 @@
+# Purpose: Run a secret scanner against the repo.
+
+name: Run Secret Scan
+
+on:
+  push:
+    branches:
+      - 840-research-third-party-secret-scanners
```

Comment:
```suggestion
      - *-secret-scanners
```

Reply:
I removed the entire "branches" section, as the only reason for it was for testing.